documentation/modules/about-cold-warm-migration.adoc

```diff
@@ -47,3 +47,83 @@ The precopy stage runs until the cutover stage is started manually or is schedul
 The VMs are shut down during the cutover stage and the remaining data is migrated. Data stored in RAM is not migrated.
 
 You can start the cutover stage manually by using the {project-short} console or you can schedule a cutover time in the `Migration` manifest.
+
+
+[id="warm-migration-versus-cold-migration"_{context}]
+== Advantages and disadvantages of cold and warm migrations
+
+[id="warm-cold-migration-overview_{context}"]
+=== Overview
+
+Both cold migration and warm migration have advantages and disadvantages, as described in the table that follows:
+
+[cols="1,1,1",options="header"]
+.Advantages and disadvantages of cold and warm migrations
+
+|===
+| |Duration |VM downtime
+
+|Cold migration
+|Minimal
+|High
+
+|Warm migration
+|Longer and variable
+|Minimal
+|===
+
+[id="warm-cold-migration-detailed_{context}"]
+=== Detailed description
+
+The table that follows offers a more detailed description of the advantages and disadvantages of each type of migration. It assumes that you have installed Red Hat Enterprise Linux (RHEL) 9 on the {ocp} platform on which you installed {project-short}.
+
+[cols="1,1,1",options="header"]
+.Detailed description of advantages and disadvantages
+
+|===
+| |Cold migration |Warm migration
+
+|*Duration*
+| Each VM is copied only once, making the duration of the migration shorter.
+| A VM might be copied in more than one snapshot. {project-short} ensures that only the most recent snapshot is used, but eliminating duplicate snapshots adds time to the duration of the migration.
```

Comment:
This might imply that we are copying the whole disk multiple times but we transfer the disk only once and then we transfer the changed blocks which are made on top of the previous snapshot. This is the main advantage of the warm migration. We don't need to transfer everything after VM is shut down. 

How about:
```
For each VM, MTV creates a snapshot which is transferred. After the cutover, MTV creates a final snapshot which contains only changed blocks between the previous snapshot. This reduces the downtime because the MTV transfers only changes not the whole disks. 
```
wdyt @RichardHoch @fabiand ?

Reply:
@mnecas Fine with me. @fabiand WDYT?

Reply:
Can we maybe simplify it:

> Each disk of a VM is first copied once, and then partially transferred again to compensate for change blocks while the VM keeps running leading to a longer and varying duration for the migration.